### PR TITLE
Ansible

### PR DESCRIPTION
**Pull request by Ranjandas**
Create new role common
Create new Makefile

### CODE REVIEW

**Comment by Ranjandas** [comment]
This role `common` is created using `ansible-galaxy --offline init common`. Which gives me all the templates.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.PHONY: all
+all:
+
+.PHONY: info
```
Comment (sebinjohn) [inline Makefile:4]: Did you intentionally left out `vagrant up` section?

File: ansible/roles/common/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+galaxy_info:
+  author: your name
```
Comment (sebinjohn) [inline ansible/roles/common/meta/main.yml]: Fill up these details?

File: Makefile (hunk 1 above)
Comment (Ranjandas) [inline Makefile:4]: Yea I missed that out. Will update it.

**Comment by sebinjohn** [comment]
Should we use `ansible-galaxy` comand to create modules? Should we make it a rule?

**Comment by Ranjandas** [comment]
@sebinjohn I think using ansible-galaxy to create roles will gives us a neat skeleton for the files in the repo. Its a great way to bootstrap ansible roles I believe. If you agree we can make it a standard.

**Comment by sebinjohn** [comment]
I like the idea. Lets go for it.

**Pull request merged**